kicad-mac-builder/CMakeLists.txt

```diff
@@ -27,13 +27,11 @@ set( KICAD_INSTALL_DIR ${CMAKE_BINARY_DIR}/kicad-dest )
 set( KICAD_CMAKE_ARGS -DDEFAULT_INSTALL_PATH=/Library/Application\ Support/kicad -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOS_MIN_VERSION} -DwxWidgets_CONFIG_EXECUTABLE=${wxwidgets_INSTALL_DIR}/bin/wx-config  -DKICAD_SCRIPTING=ON  -DKICAD_SCRIPTING_MODULES=OFF  -DKICAD_SCRIPTING_WXPYTHON=OFF  -DPYTHON_EXECUTABLE=${PYTHON_INSTALL_DIR}/bin/python2.7  -DPYTHON_INCLUDE_DIR=${PYTHON_INSTALL_DIR}/include/python2.7/ -DPYTHON_LIBRARY=${PYTHON_INSTALL_DIR}/lib/libpython2.7.dylib  -DPYTHON_SITE_PACKAGE_PATH=${wxwidgets_INSTALL_DIR}/lib/python2.7/site-packages -DCMAKE_INSTALL_PREFIX=${KICAD_INSTALL_DIR} -DKICAD_SPICE=ON -DKICAD_USE_OCE=ON -DCMAKE_BUILD_TYPE=Release )
 
 # find OCE
-execute_process(COMMAND find /usr/local/Cellar/oce -name Resources OUTPUT_VARIABLE OCE_DIR RESULT_VARIABLE FOUND_OCE)
-
-if (NOT($(FOUND_OCE) EQUAL 0))
+execute_process(COMMAND find /usr/local/Cellar/oce -name Resources OUTPUT_VARIABLE OCE_DIR RESULT_VARIABLE FOUND_OCE_EXIT_CODE)
```

Comment:
You probably want to do something similar to what was done in the old scripts here:
https://github.com/wayneandlayne/KiCadMacOSPackaging/commit/9ec546904b93c69475d637ae69e00e35a26f02ca